### PR TITLE
Add newsletter homepage

### LINKED ISSUE #195: Add link to subscribe to the newsletter in home page

### LINKED ISSUE #198: Add a button to subscribe to a dataset
Relates to #197

### PR DESCRIPTION
**Pull request by l-vincent-l**
fix #195 
image

### CODE REVIEW

File: lib/transport_web/templates/page/index.html.eex [hunk 1]
```diff
@@ -1,22 +1,46 @@
...
+                <%= dgettext("page-index", "Reuse data") %>
+                </a>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="mailing-list">
+    <div class="mailing-list__container">
+        <strong>
+            <%= dgettext("page-index", "Subscribe to the newsletter to receive update") %>
```
Comment (maukoquiroga) [inline lib/transport_web/templates/page/index.html.eex]: Subscribe to our email newsletter to receive updates

File: lib/transport_web/templates/page/index.html.eex [hunk 2]
```diff
@@ -1,22 +1,46 @@
...
+        </strong><br/>
+        <form action="//gouv.us13.list-manage.com/subscribe/post?u=5ee8bfe0f1b073b49de06a063&amp;id=13db2e9a94" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
```
Comment (maukoquiroga) [inline lib/transport_web/templates/page/index.html.eex]: Shouldn't we add ```u``` and ```id``` to an environment variable, and pass them via ```webpack```?

Comment (l-vincent-l) [reply lib/transport_web/templates/page/index.html.eex]: I think it'll be better to add the url.

**Comment by l-vincent-l** [comment]
Done.

**Pull request merged**